pkg/driverbuilder/builder/templates/ubuntu.sh

```diff
@@ -22,6 +22,7 @@ tar -xf data.tar.*
 {{end}}
 
 cd /tmp/kernel-download/usr/src/
+ls -altr
```

Comment:
Is this needed?

Reply:
I added it for useful debugging since it will only show in debug mode. I can remove it if you'd like. It would just have to be added back for debugging again later haha

Reply:
Oh I thought it might've just been mistakenly left over. Makes sense to have it in there.

Reply:
Example:
```
DEBU .+ cd /tmp/kernel-download/usr/src/
DEBU + ls -altr
DEBU �total 16
DEBU drwxr-xr-x 24 root root 4096 Oct  5  2020 linux-headers-5.4.0-51
DEBU drwxr-xr-x  4 root root 4096 Oct  5  2020 ..
DEBU drwxr-xr-x  4 root root 4096 Oct  5  2020 .
DEBU drwxr-xr-x  7 root root 4096 Oct 19 21:07 linux-headers-5.4.0-51-generic
DEBU =++ find . -type d -name 'linux-headers*generic'
```
Tells you what's actually in the folder before the `find` runs... So if `find` doesn't work, you can at least see what was actually there in debug mode.